test/active_model_serializers/test/schema_test.rb

```diff
@@ -1,5 +1,7 @@
 require 'test_helper'
 
+Rails.application.config.filter_parameters += [:password]
```

Comment:
@leonelgalan so, this is enough to make the test fail? (this should be set in the test app, I think)